inc/trilogy/protocol.h

```diff
@@ -10,10 +10,10 @@
 #include <string.h>
 
 #define TRILOGY_CAPABILITIES(XX)                                                                                       \
-    XX(TRILOGY_CAPABILITIES_NONE, 0)                                                                                   \
     /* Not used. This flag is assumed by current servers.                                                              \
```

Comment:
This seems a bit odd describing `TRILOGY_CAPABILITIES_NONE` (which is the absence of capabilities rather than a flag). Maybe this should be removed?

Reply:
Yep, sorry that's my bad. I read "assumed by current servers" and thought it was describing this as "the absence of capabilities assumed by current servers as the default", and thought maybe it was intended to describe `TRILOGY_CAPABILITIES_NONE` instead of `TRILOGY_CAPABILITIES_LONG_PASSWORD` below. I'm realizing that doesn't really make sense 🤦‍♀️